docs/self-managed/installation-methods/helm/configure/registry-and-images/install-bitnami-enterprise-images.md

```diff
@@ -58,3 +63,21 @@ helm install camunda camunda/camunda-platform --version $HELM_CHART_VERSION \
 ```
 
 This deploys Camunda with vendor-supported enterprise images, recommended for secure and stable production environments.
+
+## Understanding CVEs in Bitnami images
+
+Customers may notice a high number of CVEs when scanning Bitnami images. This is expected and does not necessarily indicate a higher security risk. The reason is that Bitnami images bundle the application (e.g., PostgreSQL, Elasticsearch, Keycloak), a full OS layer, and supporting libraries. Vulnerability scanners therefore report CVEs across all of these components, including many that are low-severity, informational, or not exploitable in practice.
+
+[Bitnami’s Open CVE Policy](https://docs.bitnami.com/kubernetes/open-cve-policy/) ensures that fixable CVEs are addressed when upstream patches become available, but open/unfixable CVEs remain until resolved by the operating system or application maintainers. As a result, even enterprise Bitnami images will continue to show high CVE counts, despite being patched against critical vulnerabilities.
+
+For Camunda users, the most important considerations are:
+
+- Bitnami subcharts are intended for development and testing convenience only.
```

Comment:
Could we remove this, please? We advise customers already that they can use in production, and we can not pivot that hard without an announcement.

Reply:
Maybe rephrased as:
Bitnami subcharts are best used in development and testing environments, unless your operational teams have expertise with Bitnami charts production deployments.

Reply:
Okay - adjusted it. I got this from: https://github.com/camunda/camunda-docs/pull/6684#discussion_r2344171454

And will also align the communication in this guide then 🫡